medusa/backup_cluster.py

```diff
@@ -173,9 +173,10 @@ def _build_backup_cmd(self):
         enable_md5_checks_option = '--enable-md5-checks' if self.enable_md5_checks else ''
 
         # Use %s placeholders in the below command to have them replaced by pssh using per host command substitution
-        command = 'mkdir -p {work}; cd {work} && medusa-wrapper sudo medusa -vvv backup-node ' \
+        command = 'mkdir -p {work}; cd {work} && medusa-wrapper {sudo} medusa -vvv backup-node ' \
                   '--backup-name {backup_name} {stagger} {enable_md5_checks} --mode {mode}' \
             .format(work=self.work_dir,
+                    sudo='sudo' if medusa.utils.evaluate_boolean(self.config.cassandra.use_sudo) else '',
```

Comment:
Do we need this during backups? IIRC we're not using `sudo` commands for backups.

Reply:
I replaced all hard-coded `sudo` occurrences with `use_sudo` config. You mean that even the hard-coded `sudo` was not required?
```
... && medusa-wrapper sudo medusa -vvv backup-node
```

Reply:
You're right actually, we need to get access to the snapshot files which could require `sudo`.